group_badges.go

```diff
@@ -0,0 +1,206 @@
+package gitlab
+
+import (
+	"fmt"
+	"net/url"
+)
+
+// GroupBadge represents a group badge.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/group_badges.html
+type GroupBadge struct {
+	ID               int    `json:"id"`
+	LinkURL          string `json:"link_url"`
+	ImageURL         string `json:"image_url"`
+	RenderedLinkURL  string `json:"rendered_link_url"`
+	RenderedImageURL string `json:"rendered_image_url"`
+	// Kind represents a project badge kind. Can be empty, when used PreviewProjectBadge().
+	Kind string `json:"kind"`
+}
+
+// GroupBadgesService handles communication with the group badges
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/group_badges.html
+type GroupBadgesService struct {
+	client *Client
+}
```

Comment:
We generally place the service type on top of the file as the first declaration.

Reply:
agreed. I've moved it to the top.